book.rb

```diff
@@ -0,0 +1,17 @@
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(date, person, book)
+    @rentals << Rental.new(date, person, book)
+  end
```

Comment:
- [ ] [OPTIONAL] While creating rental in `class Book`. we don't require to pass  `book` here. kindly fix it .
- for ex:- 
```
 def add_rental(person, date)
    # *the book itself should be sent as a parameter to create the new rental achieving the 'has-many' association. 
    Rental.new(date, self, person)
  end
```